openedx/core/djangoapps/schedules/tests/test_content_highlights.py

```diff
@@ -66,6 +68,24 @@ def test_flag_enabled(self):
             highlights,
         )
 
+    @override_waffle_flag(COURSE_UPDATE_WAFFLE_FLAG, True)
+    def test_highlights_methods_fail_gracefully_when_highlights_are_disabled(self):
+        highlights = [u'A test highlight.']
+        with self.store.bulk_operations(self.course_key):
+            self._create_chapter(highlights=highlights)
+
+        self.course.highlights_enabled_for_messaging = False
+        self.store.update_item(self.course, self.user.id)
```

Comment:
Nit: Move the above 2 lines within the `bulk_operations` context manager.  Rationale: (1) the store is updated only once at the end of the context, (2) for readability, since all modulestore updates would be within that context.